# Accumulates all the logs in a one message type

std_msgs/Header header

MsgHeader msg_header

Range range_log

SatXYZ sat_log

TrackStat track_log

